### PR TITLE
Sync old site ios

### PR DESCRIPTION
**Pull request by louquillio**

### CODE REVIEW

File: native-code/ios/index.md [hunk 1]
```diff
@@ -9,41 +9,49 @@ crumb: iOS
...
   1. Install [prerequisite software][1]
 
   2. Set the target OS in your environment:
 
      ~~~~~ bash
-     export GYP_DEFINES="OS=ios"
+     export GYP_CROSSCOMPILE=1
+     export GYP_DEFINES="OS=ios target_arch=arm"
+     export GYP_GENERATOR_FLAGS="output_dir=out_ios"
+     export GYP_GENERATORS=ninja
```
Comment (kjellander-unity) [inline native-code/ios/index.md:29]: Replace line 26-29 with just:
export GYP_DEFINES="OS=ios"

File: native-code/ios/index.md [hunk 2]
```diff
@@ -9,41 +9,49 @@ crumb: iOS
...
-### Compiling the code
+### Compiling the Code
 
 GYP is used to generate build instructions for ninja from the relevant .gyp files. Ninja is used to compile the source using the previously generated instructions. In order to configure GYP to generate build files for iOS certain environment variables need to be set. Those variables can be edited for the various build configurations as needed.
 
-Building for iOS device:
+Building for iOS Device:
 
 ~~~~~ bash
 export GYP_CROSSCOMPILE=1
```
Comment (kjellander-unity) [inline native-code/ios/index.md:57]: Replace 57-60 with:
export GYP_DEFINES="OS=ios target_arch=arm"
export GYP_GENERATOR_FLAGS="output_dir=out_ios"

File: native-code/ios/index.md [hunk 3]
```diff
@@ -57,11 +65,11 @@ Building for 64-bit iOS device:
 As above, except with:
```
Comment (kjellander-unity) [inline native-code/ios/index.md:65]: Remove line 65.

File: native-code/ios/index.md [hunk 4]
```diff
@@ -57,11 +65,11 @@ Building for 64-bit iOS device:
...
 ~~~~~ bash
-export GYP_DEFINES="OS=ios target_arch=arm64 target_subarch=arm64"
+export GYP_DEFINES="OS=ios target_arch=arm64"
 export GYP_GENERATOR_FLAGS="output_dir=out_ios64"
 ~~~~~
 
-Building for simulator:
+Building for Simulator:
 
 As above, except with:
```
Comment (kjellander-unity) [inline native-code/ios/index.md:74]: Remove line 74

File: native-code/ios/index.md [hunk 5]
```diff
@@ -70,6 +78,15 @@ export GYP_DEFINES="OS=ios target_arch=ia32"
 export GYP_GENERATOR_FLAGS="output_dir=out_sim"
 ~~~~~
 
+Building for 64-bit Simulator:
+
+As above, except with:
```
Comment (kjellander-unity) [inline native-code/ios/index.md:83]: Remove line 83

File: native-code/ios/index.md [hunk 6]
```diff
@@ -70,6 +78,15 @@ export GYP_DEFINES="OS=ios target_arch=ia32"
...
+
+~~~~~ bash
+export GYP_DEFINES="OS=ios target_arch=x64"
+export GYP_GENERATOR_FLAGS="output_dir=out_sim"
+~~~~~
+
 Building for OSX:
 
 As above, except with:
```
Comment (kjellander-unity) [inline native-code/ios/index.md:92]: Remove line 92.

**Comment by kjellander-unity** [comment]
Thanks for this, I only reviewed the ios page in this PR.
I hope you don't mind me making some corrections now that we're at it?

**Comment by louquillio** [comment]
Done, in 6b41ba0

**Pull request merged**